src/core/sys/posix/stdio.d

```diff
@@ -344,3 +344,7 @@ unittest
     assert(memcmp(ptr, testdata.ptr, testdata.length*wchar_t.sizeof) == 0);
     assert(fclose(f) == 0);
 }
+
+
+ssize_t getdelim (char** lineptr, size_t* n, int delimiter, FILE* stream);
+ssize_t getline (char** lineptr, size_t* n, FILE* stream);
```

Comment:
No, `scope char** lineptr` is correct here, the function doesn't escape the value of the `char**` pointer passed in. In any case it hardly seems to matter b/c those functions are inherently unsafe, after the call `*lineptr` will point to some internal buffer of the stream.